.stylelintrc.yml

```diff
@@ -0,0 +1,65 @@
+extends:
+  - stylelint-config-standard
+
+plugins:
+  - "@double-great/stylelint-a11y"
+
+rules:
+  # Suppress stylelint-config-recommended errors
```

Comment:
I don't think we should have all those exception but instead we should work toward to remove all of them and fix our code base to match as close as possible to apply all the standard recommendation set of rules.

I do know the immediate need with this feature. I am proposing to merge this one after this is completed
* Add an explicit comment in our config regarding the task of removing the stylelint rule exception
* Create a new Github issue to perform the clean up exercise to remove the stylelint rule exception in our config or to add a justification beside of each rule explaining why we do have such exception on.

Reply:
The immediate need for this PR is to open a path forward for #9529. It fixes a 3 month old issue reported in #9503 and impacts all Twitter timeline widgets.

Csslint fails on it and there's nothing I can do to get around it without either scrapping or replacing it. Other options like using ignore comments aren't feasible because csslint's ignore syntax is inherently flawed (doesn't unconditionally ignore - it still fails in my scenario). Excluding WET's main compiled CSS file from the ``csslint`` task isn't practical either since not linting that file would negate the task's purpose.

---

Regarding exemptions, I agree that there should be plenty of room to update WET's SCSS/CSS to better align with ``stylelint-config-standard``'s ruleset and enable more of ``stylelint-a11y``'s rules. But I don't think enough pieces of the puzzle are in place yet to start tackling those.

As mentioned in https://github.com/wet-boew/wet-boew/pull/9539#issuecomment-1454119035, I expect that this'll need to be tackled in phases across a few PRs. IMO doing it all at once would be very messy and time-consuming.

IMO it also probably won't be realistic to fully follow ``stylelint-config-standard``'s ruleset nor activate all of ``stylelint-a11y``'s rules. Following some of the former's rules would guarantee breaking IE11 (which I'm personally on board with - but I don't think PP is yet) and shift indentation styles to spaces instead of tabs. Some of the latter's rules are opinionated and IMO non-starters for WET.

Reply:
I am not saying to hold on, but to open a new issue to ensure we don't forget about it with a comments in that new config file to add emphasis that we are working on it and a reference (the new issue number) where we can track progress and discussion.

@EricDunsworth Do you want me to create that new issue or you will do?

Reply:
See my another request of change.